crates/sui-protocol-config/src/lib.rs

```diff
@@ -2479,7 +2481,15 @@ impl ProtocolConfig {
                             PerObjectCongestionControlMode::TotalTxCount;
                     }
 
+                    // Enable Mysticeti on mainnet.
                     cfg.feature_flags.consensus_choice = ConsensusChoice::Mysticeti;
+
+                    // Enable leader scoring & schedule change on mainnet for mysticeti.
+                    cfg.feature_flags.mysticeti_leader_scoring_and_schedule = true;
+
+                    // Enable checkpoint batching on mainnet.
+                    cfg.checkpoint_summary_version_specific_data = Some(1);
+                    cfg.min_checkpoint_interval_ms = Some(200);
                 }
                 53 => {
```

Comment:
since 52 was reverted from testnet branch, do we need to introduce 53 right now? Or should we collapse them?

Reply:
Yeah good point, 52 & 53 would get released together anyways so I guess there is no harm in collapsing them into 52.